engine/org.eclipse.birt.report.engine/src/org/eclipse/birt/report/engine/emitter/IContentEmitter.java

```diff
@@ -123,4 +123,6 @@
 	void startListGroup( IListGroupContent group ) throws BirtException;
 
 	void endListGroup( IListGroupContent group ) throws BirtException;
+	
+	boolean isEmitterEnableMultipleSheet( );
```

Comment:
Can we change name to "isMultipleSheetsEnabled" and add comments for this method?